tools/codegen/gen.py

```diff
@@ -168,15 +168,31 @@ def cpp_string(s: str) -> str:
 # code we want.
 Target = Enum('Target', ('DEFINITION', 'DECLARATION', 'REGISTRATION'))
 
-# Dispatch keywords in native_functions.yaml that support all backends.
-KEYWORD_ALL_BACKENDS = ('DefaultBackend', 'Math')
+# Dispatch keys that "support all backends".  These codegen slightly differently
+# then backend specific keys.
+def is_generic_dispatch_key(dk: str) -> bool:
+    return dk in {'DefaultBackend', 'Math'}
```

Comment:
Personally I'd rather retain a top-level definition for this than sink it into a helper like this (though _adding_ a helper that tests it is fine by me) - feels fragmented to have part of the data model hidden in a function.

Reply:
I'll leave this to future work

Reply:
> I'll leave this to future work

What, putting the top-level definition back? 😁 

I probably wasn't clear - what I meant was, the function you added is fine by me, but I'd rather see it use the existing top-level definition, rather than capture it. Like:
```
GENERIC_DISPATCH_KEYS = ('DefaultBackend', 'Math')
def is_generic_dispatch_key(dk: str) -> bool:
    return dk in GENERIC_DISPATCH_KEYS
```

Reply:
Yeah, haha (because I had a clean land bill so it was ready to land).

I do understand the suggestion here... it just conflicts with another pref I have, which is default lots of encapsulation (hide everything by default) and then make it visible. But it's NBD, and there's more refactoring for Dispatch Keys that probably could be done (like making it into an enum)